src/components/SideMenu/SideMenu.test.tsx

```diff
@@ -23,32 +23,32 @@ describe("Test side-menu", () => {
     expect(menuComponent).toBeInTheDocument();
   });
 
-  test("should change selected item on click", () => {
+  test("should change selected item on click", async () => {
     render(
       <MemoryRouter
         initialEntries={[
           {
-            pathname: routes.HOME,
+            pathname: routes.ADMIN_HOME,
           },
         ]}
       >
         <AdminHomePage />
```

Comment:
I think you can render only `SideMenu` component, because like this you are rendering AdminHomePage and when this page is rendered `books-button` shouldn't be displayed

Same for other tests here that tests if buttons are selected